pkgs/by-name/ya/yaak/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  appimageTools,
+  targetPlatform,
```

Comment:
```suggestion
  hostPlatform,
```
and same below.

Reply:
Are you sure about this? On a previous derivation I made, I received a review suggesting to use `targetPlatform` so that the derivation is built according to the target inferred / manually supplied which is that place where package is supposed to be run - am I wrong?

Reply:
So the basic idea is:
- buildPlatform = where the package is built (compiled)
- hostPlatform = where the package is run
- targetPlatform = platform for which the package can compile code

This is not a compiler, so it should not care about the targetPlatform at all. It is unfortunate that basically every other tool calls the host platform the "target".

Reply:
That was super helpful, thanks! will push the change in a while.